sql/migrations/user-collection/up.sql

```diff
@@ -0,0 +1,30 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS bookbrainz.user_collection (
+	id UUID PRIMARY KEY DEFAULT public.uuid_generate_v4(),
+	editor_id INT,
+	name VARCHAR(80) NOT NULL CHECK (name <> ''),
+	description TEXT NOT NULL DEFAULT '',
+	type bookbrainz.entity_type NOT NULL,
+	public BOOLEAN NOT NULL DEFAULT FALSE,
+	created_at TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT timezone('UTC'::TEXT, now()),
+   	last_modified TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT timezone('UTC'::TEXT, now())
```

Comment:
Small nitpick: this line isn't aligned

Reply:
i was using spaces instead of tab here :P